source/get-started/install-configfile.rst

```diff
@@ -131,54 +125,51 @@ Perform the following steps:
 
 .. _install-configfile-yaml-end:
 
-Start the installation with the command:
+#. Start the unattended installation using the `--config` option.

```

Comment:
This step need to be left-aligned to the page. It shouldn't restart numbering at 1. 

Also, I recommend adding a line to indicate that the user should complete installation.